### PR TITLE
OpenShift fixes catched by real testing

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request brings couple fixes catched by real testing.
The changes affected only OpenShift.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -103,5 +103,5 @@ def get_dir(system_path=None, virtual_path=None):
         'Topic :: Software Development',
     ],
     install_requires=open('requirements.txt').read().splitlines(),
-    zip_safe=True
```
Comment (TomasTomecek) [inline setup.py]: I thought it was set to True to fix a packaging issue.

File: moduleframework/helpers/openshift_helper.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def _app_exists(self):
         if int(oc_status.exit_status) == 0:
             common.print_info("Application already exists.")
             return True
-        oc_services = self.runHost("oc get services -o json", ignore_status=True).stdout
+        oc_services = self.runHost("oc get pods -o json", ignore_status=True).stdout
```
Comment (TomasTomecek) [inline moduleframework/helpers/openshift_helper.py]: this seem odd, are you sure you know what you are doing?

Comment (phracek) [reply moduleframework/helpers/openshift_helper.py]: I will clarify it. `pods` make sense, but what if application failed. And the other use cases.

File: moduleframework/avocado_testers/avocado_test.py [hunk 3]
```diff
@@ -226,7 +226,7 @@ def copyFrom(self, *args, **kwargs):
         """
         return self.backend.copyFrom(*args, **kwargs)
 
-    def getIPaddr(self, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (phracek) [reply moduleframework/avocado_testers/avocado_test.py]: I don't see any reason, why to use args or kwargs.

File: setup.py (hunk 1 above)
Comment (phracek) [reply setup.py]: My fault. You are right.

File: moduleframework/avocado_testers/avocado_test.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (jscotka) [reply moduleframework/avocado_testers/avocado_test.py]: I mean other methods in this class like: getModuleYaml and more: https://github.com/phracek/meta-test-family/blob/46f542b7c9678544d53dc2bf1d2d9b54966e3093/moduleframework/avocado_testers/avocado_test.py#L188 
almost all methods contais this pass thru arguments and probably would be nice to reconsider it and remove all these param forwarding and have there explicit params (to be easier to use also code completion)

File: moduleframework/helpers/openshift_helper.py (hunk 2 above)
Comment (phracek) [reply moduleframework/helpers/openshift_helper.py]: Based on discussion with OpenShift folks, `oc get pods` is better for getting status if application is READY.

**Comment by phracek** [comment]
@jscotka Yes, but I have to run `oc cluster up`.  And unfortunally OpenShift server is not on PyPi.
I am trying to solve it by separate PR. https://github.com/fedora-modularity/meta-test-family/pull/195

**Pull request merged**